tests/data/config/l.py

```diff
@@ -8,3 +8,4 @@ def func(x):
 _base_ = ['./l1.py', './l2.yaml', './l3.json', './l4.py']
 item3 = False
 item4 = 'test'
+item5 = osp.expanduser('~')
```

Comment:
Hi, Thanks for your contribution! `test_merge_from_multiple_bases` should also be modified.